saleor/graphql/account/mutations/account.py

```diff
@@ -408,12 +411,12 @@ def perform_mutation(cls, _root, info, **data):
             raise ValidationError(
                 {"redirect_url": error}, code=AccountErrorCode.INVALID
             )
-        token_kwargs = {
+        token_payload = {
             "old_email": user.email,
             "new_email": new_email,
             "user_pk": user.pk,
         }
-        token = create_jwt_token(token_kwargs)
+        token = create_token(token_payload, timedelta(hours=1))
```

Comment:
Wondering whether we shouldn't extract this `timedelta(hours=1)` to some setting? I think you may want to customize for how long your links are valid and now this is hidden in code.

Reply:
It was taken from the previous implementation. I didn't change it and I used the same time delta as I thought that there could be some agreements.  Of course, I will change it as you said.
👮‍♂️